index.html

```diff
@@ -1262,13 +1262,23 @@ <h2>Core Vocabulary Definitions</h2>
       <section id="sec-data-schema-vocabulary-definition">
       <h2>Data Schema Vocabulary Definitions</h2>
       <p>
-        The data schema definition is reflecting a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]]. It is noted
+        The data schema vocabulary definition is reflecting a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]]. It is noted
         that data schema definitions within Thing Description instances are not limited to this defined subset and may use additional terms
         found in JSON Schema using a <a>TD Context Extension</a> for the additional terms as described in
         <a href="#sec-context-extensions"></a>, otherwise these terms are semantically ignored by <a>TD Processors</a>
         (for details about semantic processing, please refer to the documentation under the namespace IRIs, e.g.,
         <a href="https://www.w3.org/2019/wot/td">https://www.w3.org/2019/wot/td</a>).
       </p>
+      <p>A data schema is an abstract notation for data contained in data formats. 
+         In a TD, concrete data formats are specified in Forms (see <a href="#form"></a>) using content types.
+         When the value of a content type in an instance of the Form is <code>application/json</code>, 
+         the data schema can be processed directly by JSON Schema processors.
+         Otherwise, Web of Things (WoT) Binding Templates [[?WOT-BINDING-TEMPLATES]] defines data schema's 
+         available mappings to other content types such as XML [[?xml]]. 
+         If the content type in an instance of the Form is not <code>application/json</code> and 
```

Comment:
lets change "... <code>application/json</code> and ..." to "... <code>application/json</code> and if ..."